example/fly_mission/CMakeLists.txt

```diff
@@ -6,34 +6,12 @@ if(NOT MSVC)
     add_definitions("-std=c++11 -Wall -Wextra -Werror")
 else()
     add_definitions("-std=c++11 -WX -W2")
-    set(platform_libs "Ws2_32.lib")
 endif()
 
-# Add DEBUG define for Debug target
-set(CMAKE_CXX_FLAGS_DEBUG "-DDEBUG")
-
-# This finds thread libs on Linux, Mac, and Windows.
-find_package(Threads REQUIRED)
-
-# Not needed if DroneCore installed system-wide
-include_directories(
-    ${CMAKE_SOURCE_DIR}/../../install/include
-)
-
 add_executable(fly_mission
     fly_mission.cpp
 )
 
-# Not needed if DroneCore installed system-wide
```

Comment:
Does the fact this is removed mean that you have sorted out Windows case for installing system wide?

Reply:
I have currently no access to a Windows machine. I'll come back to the Windows build once I do.

Reply:
So the plan is to do that before this is merged? It doesn't "matter" except that the example docs will need to be changed to add "not supported" for windows.

Reply:
Ignore that, I see below that you're hopefully going to do that as part of this PR :-)